podcast-player/src/components/episode/EpisodeItem.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
```

Comment:
Best practice to not include it? You've mentioned this and I've seen some comments online about it but haven't seen documentation on it. And of course all my snippets include this statement so I usually leave it in there, because it was always pretty normal to have it. Also the compiler never complains about it being unused like it does everything else.

Reply:
It's a somewhat recent change, so some tooling might expect it to be there. Some tooling might not care, and the most intelligent and strict tooling will warn / error on unused imports.